### ISSUE TITLE
recalculate time to closing/opening

### ISSUE THREAD

**Issue by matkoniecz** [ypid/opening_hours_map#38]
Currently once popup is created time to closing/opening stays the same. To recalculate it user must close it and click again selected place.

It would be preferable to recalculate time till state change every 60 seconds.

**Comment by ypid** [comment]
The map is already kind of slow. This would even make it slower for low zoom levels. Maybe it makes sense for now to just reevaluate when an POI is clicked?

**Comment by matkoniecz** [comment]
Yes, I though about doing it only for POIs in the currently open popup.

For green/red/yellow rings on the map - probably overcomplicated but faster would be to keep sorted list of future state changes what would allow to avoid recalculating everything every 60 seconds.

Also I made correction to report - it is necessary to reload entire page to recalculate times.

> The map is already kind of slow.

I never noticed this problem, but in my area amount of tagged opening hours is low.

**Comment by ypid** [comment]
> sorted list of future state changes

Sounds like the solution for this :+1:

> I never noticed this problem, but in my area amount of tagged opening hours is low.

I am usually testing with major cities in Germany so it can get slow :wink:
